### ISSUE TITLE
Can't find config files 

### ISSUE THREAD

**Issue by GPrathap** [eleurent/robust-beyond-quadratic#1]
Can not find obstacle_env 

File "/usr/local/lib/python3.6/dist-packages/rl_agents-1.0.dev0-py3.6.egg/rl_agents/agents/common/factory.py", line 60, in load_environment
ModuleNotFoundError: No module named 'obstacle_env'

**Comment by eleurent** [comment]
Oh that's right, I forgot to add it to the instructions!
The environment is here: https://github.com/eleurent/obstacle-env
You can install it with ```pip install --user git+https://github.com/eleurent/obstacle-env```

I'll update the readme, thanks for the feedback!

**Comment by eleurent** [comment]
And now that I am checking, I also found out that I forgot to merge the development branch.
I will do that as soon as possible

**Comment by GPrathap** [comment]
Yeah, sorry I forgot to mention here. Yet I found obstacle-env repo and it is in dev-robust. By the way,  still, there is a small mistake in the README.txt 

I guess for some reason files were renamed, just update them as well. For example, 

python experiments.py configs/ObstacleEnv/env.json configs/ObstacleEnv/agents/robust-epc.json 

python3 experiments.py **evaluate** configs/ObstacleEnv/env.json configs/ObstacleEnv/**RobustEPC.json** **--train**


Anyway thank you so much for sharing this, it's really interesting  :)

**Comment by eleurent** [comment]
Indeed, I could not find the commits where these files were renamed... I'm quite afraid that they stayed on a local branch of my previous work computer 🤦‍♂️

But I managed to fix a few issues, it should be fine now.
Thanks a lot for your interest and investigation!
